packages/sdk-metrics/src/aggregator/ExponentialHistogram.ts

```diff
@@ -213,32 +213,33 @@ export class ExponentialHistogramAccumulation implements Accumulation {
   }
 
   /**
-   * merge combines data from other into self
-   * @param {ExponentialHistogramAccumulation} other
+   * merge combines data from previous value into self
+   * @param {ExponentialHistogramAccumulation} previous
    */
-  merge(other: ExponentialHistogramAccumulation) {
+  merge(previous: ExponentialHistogramAccumulation) {
     if (this._count === 0) {
-      this._min = other.min;
-      this._max = other.max;
-    } else if (other.count !== 0) {
-      if (other.min < this.min) {
-        this._min = other.min;
+      this._min = previous.min;
+      this._max = previous.max;
+    } else if (previous.count !== 0) {
+      if (previous.min < this.min) {
+        this._min = previous.min;
       }
-      if (other.max > this.max) {
-        this._max = other.max;
+      if (previous.max > this.max) {
+        this._max = previous.max;
       }
     }
 
-    this._sum += other.sum;
-    this._count += other.count;
-    this._zeroCount += other.zeroCount;
+    this.startTime = previous.startTime;
```

Comment:
This is the only line I added

Sorry for the noisy diff, but i wanted to rename to make it clear that the start times don't need to be compared and previous is always older (it's only called with previous in the `ExponentialHistogramAggregator.merge()`).

Reply:
Thats's fine, thanks :slightly_smiling_face: This way it's in also line with the other aggregators.